bitnami/grafana/Chart.yaml

```
@@ -24,4 +24,4 @@ name: grafana
 sources:
   - https://github.com/bitnami/bitnami-docker-grafana
   - https://grafana.com/
-version: 7.0.7
+version: 7.1.7
```

There's a typo here:
```suggestion
version: 7.1.0
```

---

Sorry for that mistake. I've corrected that.